test/lit.cfg

```diff
@@ -125,18 +125,10 @@ if config.test_exec_root is None:
     if not llvm_config:
         lit_config.fatal('No site specific configuration available!')
 
-    # Get the source and object roots.
-    llvm_src_root = lit.util.capture(['llvm-config', '--src-root']).strip()
+    # Get the object root.
     llvm_obj_root = lit.util.capture(['llvm-config', '--obj-root']).strip()
-    flang_src_root = os.path.join(llvm_src_root, "tools", "flang")
     flang_obj_root = os.path.join(llvm_obj_root, "tools", "flang")
```

Comment:
This code assumes an in-tree build, which I suspect is no longer supported or functional. Should we just clean this up? Or can you test the in-tree build process and see if it works?

Reply:
In-tree build is problematic since monorepo has appeared. I'd rather drop this feature before it start to be a serious maintenance nightmare.